app/forms/anti-affinity-group-create.tsx

```diff
@@ -0,0 +1,106 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, you can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * Copyright Oxide Computer Company
+ */
+import { useForm } from 'react-hook-form'
+import { useNavigate, type LoaderFunctionArgs } from 'react-router'
+
+import {
+  queryClient,
+  useApiMutation,
+  usePrefetchedQuery,
+  type AntiAffinityGroupCreate,
+} from '@oxide/api'
+
+import { DescriptionField } from '~/components/form/fields/DescriptionField'
+import { NameField } from '~/components/form/fields/NameField'
+import { RadioField } from '~/components/form/fields/RadioField'
+import { SideModalForm } from '~/components/form/SideModalForm'
+import { HL } from '~/components/HL'
+import { titleCrumb } from '~/hooks/use-crumbs'
+import { getProjectSelector, useProjectSelector } from '~/hooks/use-params'
+import { addToast } from '~/stores/toast'
+import { pb } from '~/util/path-builder'
+
+import { affinityGroupList, antiAffinityGroupList } from './affinity-util'
+
+export const handle = titleCrumb('New anti-affinity group')
+
+export async function clientLoader({ params }: LoaderFunctionArgs) {
+  const { project } = getProjectSelector(params)
+  await Promise.all([
+    queryClient.prefetchQuery(antiAffinityGroupList({ project })),
+    queryClient.prefetchQuery(affinityGroupList({ project })),
+  ])
+  return null
+}
```

Comment:
Waiting for these in the loader means there's a loading bar when you click the button to open the form. Probably better not to prefetch (or to kick off prefetch in the loader but not `await` it) so that the form opens instantly. The only annoying thing is you probably have to disable submit until these requests come back. The loading time shouldn't be noticeable because nobody is filling the form in under a second (except Playwright).